elementary/monitor/dbt_project/models/tests_recommendation/table_timestamp_columns.sql

```diff
@@ -76,6 +81,18 @@ with
         where loaded_at_field is not null
     ),
 
+    -- Users can provide the timestamp columns for their models,
+    -- if provided, we assign a confidence score of 0 (certain).
+    model_provided_timestamp_columns as (
+        select
+            lower(database_name) as database_name,
+            lower(schema_name) as schema_name,
+            lower(name) as table_name,
+            lower(partition_by_field) as column_name
```

Comment:
We need to make sure that this field is a `date/timestamp` since you can also partition by an integer.